src/api/root/handler.js

```diff
@@ -46,7 +36,24 @@ const healthCheck = new HealthCheck(packageJson, [
  * @param {*} request - the Hapi request object
  * @param {*} h - the Hapi handler object
  */
-const getHealth = defaultHealthHandler(healthCheck)
+const getHealth = (request, h) => {
+  const simpleHealthCheck = request.query && 'simple' in request.query && (request.query.simple === '' || request.query.simple === true)
+  let healthCheck
+  if (simpleHealthCheck) {
+    healthCheck = new HealthCheck(packageJson, [])
```

Comment:
Ironically, creating a new HealthCheck object on every `getHealth` request is much less efficient than just recycling the existing one like the old implementation.

I would suggest initializing 2 HealthCheck objects: `healthCheck` and `simpleCheck` in the global scope of this file, and then:
```js
if (simpleHealthCheck) {
  return defaultHealthHandler(simpleCheck)(request, h)
}

   return defaultHealthHandler(healthCheck)(request, h) 
```